CHANGELOG.rst

```diff
@@ -7,14 +7,26 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
+
+[1.3.4] - Unreleased
+^^^^^^^^^^^^^^^^^^^^
+
+Fixed
+-----
+- Policy ensemble no longer incorrectly wrings "missing mapping policy" when
+  mapping policy is present.
+  
+
 [1.3.3] - 2019-09-11
+
 ^^^^^^^^^^^^^^^^^^^^
 
 Fixed
 -----
 - Added a check to avoid training CountVectorizer for a particular attribute of a message if no text is provided for that attribute across the training data.
 
 
+
```

Comment:
also this one :P

Reply:
check again if the lines are good now, it's consistent with stuff lower down the file now